CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Change history for eslint-config-stripes
 
-## 3.2.0 (IN PROGRESS)
-
-* Turn off `react/destructuring-assignment` rule. Available from v3.1.1.
+## [3.2.0](https://github.com/folio-org/eslint-config-stripes/tree/v3.2.0) (2018-09-12)
+[Full Changelog](https://github.com/folio-org/eslint-config-stripes/compare/v3.1.0...v3.2.0)
+* Turn off `react/destructuring-assignment` rule.
```

Comment:
Interesting. Hadn't occurred to me that, when making a new minor release, we move to a state where downstream developers no longer need to depend on (and therefore know about) the trivial version that any of the individual features relied on. So I quite like this.

OTOH, I can also imagine reading ui-whatever, seeing that it depends on eslint-config-stripes v3.1.1, then looking at the eslint-config-stripes changelog and being a bit bemused at the lack of any mention of v3.1.1.

So on balance, I think I'd prefer to keep these in. But I don't feel strongly about it.

Reply:
3.1.1 will only exist on the `npm-folioci` registry.